cmd/v/help/self.txt

```diff
@@ -0,0 +1,8 @@
+Usage: v self
+
+Rebuild V with the passed options.
+
+Options:
+  -debug       - Display debugging info.
```

Comment:
There is no `-debug`. `v self` treats only `-o` slightly differently - it means the final name of the executable, if the self compilation was successfull, so you can do `v -o v2 self` for example.

Reply:
All other options are just passed down to the underlying `v cmd/v` .